### PR TITLE
Feat/ansible playbook

### PR DESCRIPTION
**Pull request by rainzm**
**这个 PR 实现什么功能/修复什么问题**:

1. Add support for requirements. Now you can declare the source info of the referenced role in requirements.
2. Add Roles for Play in pkg/util/ansiblev2
3. Add climc operator for ansibleplaybooks_v2: ansibleplaybooksv2-run, ansibleplaybooksv2-stop, ansibleplaybooksv2-delete
4. Install git in Dockerfile.ansibleserver-base

**是否需要 backport 到之前的 release 分支**:
NONE

/area ansibleserver
/cc @yousong @zexi

### CODE REVIEW

File: pkg/util/ansiblev2/playbook.go [hunk 1]
```diff
@@ -204,6 +205,13 @@ func (play *Play) MarshalYAML() (interface{}, error) {
 	if len(play.Vars) > 0 {
 		r["vars"] = play.Vars
 	}
+	if len(play.Roles) > 0 {
+		roles := make([]map[string]string, len(play.Roles))
+		for i := range play.Roles {
+			roles[i] = map[string]string{"role": play.Roles[i]}
+		}
+		r["roles"] = roles
```
Comment (yousong) [inline pkg/util/ansiblev2/playbook.go:213]: 建议使用ansible `include_role`, `import_role`，扩展成`IncludeRole`, `ImportRole` ITask

Ref: https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#using-roles

Comment (rainzm) [reply pkg/util/ansiblev2/playbook.go]: fix, please review again.

**Pull request merged**